src/app/shared/comment-editor/markdown-toolbar/markdown-toolbar.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input, OnInit } from '@angular/core';
+import '@github/markdown-toolbar-element';
```

Comment:
This import doesn't look the same as the usual

Reply:
I have tried the usual way of importing:
```typescript
import MarkdownToolbarElement from '@github/markdown-toolbar-element';
```
but it doesn't work:
- clicking on buttons in the toolbar has no effect on the text area.

The module declares some global interfaces and I think this importing it this way is necessary for it to run these declaration statements?